operations/iuf/workflows/image_preparation.md

```diff
@@ -29,6 +29,10 @@ Refer to that table and any corresponding product documents before continuing to
     iuf -a "${ACTIVITY_NAME}" -m "${MEDIA_DIR}" run --site-vars "${ADMIN_DIR}/site_vars.yaml" -bpcd "${ADMIN_DIR}" -r update-cfs-config prepare-images
     ```
 
+**`NOTE`:** If it is necessary to execute a `sat bootprep` command directly from management
+nodes on CSM 1.5.2 systems to debug a failure, an error may occur during image customization.
+To fix this issue, see the workaround documented in [`sat bootprep` image customization error](../../../troubleshooting/known_issues/sat_bootprep_image_customization_error.md)
```

Comment:
This "NOTE" no longer a part of the step, and this results in the numbering of the next step being reset to 1 again. I think it can be fixed by indenting these lines, although I'm seeing inconsistent use of hard line breaks and indentation in this file, so you'll have to try and see what works in the rendered version.